client-app/app/components/patterns-list.js

```diff
@@ -1,11 +1,13 @@
 import Component from "@ember/component";
-import { not } from "@ember/object/computed";
+import { not, equal } from "@ember/object/computed";
 import { computed } from "@ember/object";
 import Pattern from "client-app/models/pattern-item";
 import { ajax } from "client-app/lib/utilities";
 
 export default Component.extend({
   immutable: not("mutable"),
+  showCounter: equal("key", "suppression"),
+  showResetCounter: equal("key", "suppression"),
 
```

Comment:
I don't follow that, if `showCounter` and `showResetCounter` got same value, 
do we need 2 conditional statements?

```
    {{#if showCounter}}
      <input disabled=true title="Number of logs that have been suppressed by this pattern" class="count shrink" value={{pattern.count}}>
    {{/if}}
    {{#if showResetCounter}}
      <button {{action "resetCount" pattern}} class="btn no-text reset shrink" title="Reset count of suppressed logs" disabled={{pattern.zeroCount}}>
        <i class="fa fa-rotate-right"></i>
      </button>
    {{/if}}
```